examples/check.py

```diff
@@ -47,7 +47,23 @@
 print "======================================================="
 print "\n"
 
-# Creating a Check using the previously created bank account
+# Verifying a Bank Account with the microdeposit amounts
```

Comment:
@pon @elnaz having the verify process in the check example might be weird if you cannot verify a bank over and over again